.github/workflows/build_test.yml

```diff
@@ -158,7 +158,6 @@ jobs:
     runs-on: ubuntu-latest
     env:
       USE_WHEEL: 1
-      NPY_RELAXED_STRIDES_DEBUG: 1
```

Comment:
Decided to move this to the other one (so use_wheel is just that and nothing else) and rename that one.